frontend/packages/kubevirt-plugin/src/constants/vm/default-os-selection.ts

```diff
@@ -19,7 +22,13 @@ export class OSSelection extends ObjectEnum<string> {
 
   public getContainerImage = () => this.image;
 
-  static getAll = () => [OSSelection.FEDORA, OSSelection.CENTOS];
+  static getAll = () => {
+    if (isUpstream()) {
+      return [OSSelection.FEDORA, OSSelection.RHEL8, OSSelection.CENTOS];
+    }
+
+    return [OSSelection.RHEL8, OSSelection.FEDORA, OSSelection.CENTOS];
+  };
```

Comment:
may I suggest to switch the if statement to be with a ternary operator? something like:
static getAll = () => isUpstream() ?
[OSSelection.FEDORA, OSSelection.RHEL8, OSSelection.CENTOS]:
[OSSelection.RHEL8, OSSelection.FEDORA, OSSelection.CENTOS];

Reply:
:+1: Thanks, fixed